plugins/module_utils/network/nxos/config/lag_interfaces/lag_interfaces.py

```diff
@@ -286,8 +286,8 @@ def add_commands(self, diff, name):
             commands.append("interface" + " " + d["member"])
             cmd = ""
             group_cmd = "channel-group {0}".format(name)
-            if "force" in d:
-                cmd = group_cmd + " force " + d["force"]
+            if "force" in d and d["force"]:
```

Comment:
Is this not sufficient?
```suggestion
            if d.get("force"):
```

Reply:
> Is this not sufficient?

Yes, but I'm worried about the existence of the keyword "force" in the dictionary.

Maybe is better? :

`if d.get("force",False):`

Reply:
> Yes, but I'm worried about the existence of the keyword "force" in the dictionary.

That's what `.get()` is for? The default action is to return `None` when the key doesn't exist, which is falsy and so fails the conditional. Unless you're explicitly testing for `False`, which is unnecessary here, there's no reason to set the default value to anything else.

Reply:
Thank you, sorry I'm pretty new to this world.